src/actions/arcActions.ts

```diff
@@ -327,6 +327,7 @@ async function getProposalDetails(daoInstance: Arc.DAO, votingMachineInstance: A
     beneficiaryAddress: contributionProposal.beneficiaryAddress.toLowerCase(),
     boostedTime,
     boostedVotePeriodLimit,
+    quietPeriod: votingMachineParams.quietEndingPeriod,
```

Comment:
we dont need this

Reply:
How would you know the length of the quiet ending then?

Reply:
because currentBoostedVotePeriodLimit gets set to it by Arc and we update proposal.boostedVotePeriodLimit on our end to match it (we should really match all the names)